changelogs/unreleased/gh-10485-fix-user-DDL-memleak.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/box
+
+- Fixed a memory leak on user DDL when access is denied (gh-10485).
```

Comment:
Bad wording: DDL is "data definition language" so the sentence doesn't make any sense. Should be something like "Fixed a memory leak on an attempt to create or modify a user denied due to access restrictions". The doc team may suggest a better variant.

Reply:
Fixed as suggested.